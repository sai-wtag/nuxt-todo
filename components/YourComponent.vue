<template>
  <div></div>
</template>
<script>
export default {
  name: 'YourComponent',
  data() {
    return {
      // ...
    }
  },
  methods: {
    // ...
  },
}
</script>
