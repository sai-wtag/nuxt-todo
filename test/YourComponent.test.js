import { mount } from '@vue/test-utils'
import YourComponent from '@/components/YourComponent'

describe('YourComponent', () => {
  test('renders correctly', () => {
    const wrapper = mount(YourComponent)
    expect(wrapper.element).toMatchSnapshot()
  })
  test('is a Vue instance', () => {
    const wrapper = mount(YourComponent)
    expect(wrapper.vm).toBeTruthy()
  })
})
