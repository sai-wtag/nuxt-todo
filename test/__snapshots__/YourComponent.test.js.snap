// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`YourComponent renders correctly 1`] = `<div />`;
